docker-compose.yml

```diff
@@ -188,25 +188,24 @@ services:
     volumes:
       - trino-postgres:/data
 
-  hive-metastore-mariadb:
-    image: mariadb:10.11.5
```

Comment:
Switched this to postgres, as the starburst hive image uses some metastore storage type that isn't supported in mariadb.